charts/sps-sample/README.md

```
@@ -5,5 +5,5 @@
 | Name            | Description                                                                                            | Value |
 |-----------------|--------------------------------------------------------------------------------------------------------|-------|
 | refarch-gateway | Config for refarch-gateway see https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway | {}    |
-| frontend        | Config for frontend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/frontend    | {}    |
-| backend         | Config for frontend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/backend     | {}    |
+| frontend        | Config for frontend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/charts/frontend    | {}    |
+| backend         | Config for frontend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/charts/backend     | {}    |
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)